tools/V4-CSDL-to-OpenAPI.xsl

```diff
@@ -3862,8 +3862,9 @@
         <xsl:text>time'</xsl:text>
       </xsl:when>
       <xsl:when test="$type='Edm.Int64' or $type='Edm.Int32' or $type='Edm.Int16' or $type='Edm.SByte' or $type='Edm.Byte' 
-                   or $type='Edm.Double' or $type='Edm.Single' or $type='Edm.Date' or $type='Edm.DateTimeOffset' or $type='Edm.Guid' or $type='Edm.TimeOfDay'" />
-      <!-- TODO: handle other Edm types, enumeration types, and type definitions -->
+                   or $type='Edm.Decimal' or $type='Edm.Double' or $type='Edm.Single' or $type='Edm.Boolean' 
+                   or $type='Edm.Date' or $type='Edm.DateTimeOffset' or $type='Edm.Guid' or $type='Edm.TimeOfDay'" />
+      <!-- TODO: handle enumeration types and type definitions -->

```

Comment:
What about `Edm.Duration`? In OData 4.0, this requires the prefix `duration`, and this prefix is also accepted by OData 4.01. But this transformation omits the prefix and therefore supports only 4.01.

Reply:
Which flavors does RAP accept?

Reply:
Currently only with `duration` prefix